test/bin/common.sh

```diff
@@ -35,11 +35,11 @@ NEXT_RPM_SOURCE="${ROOTDIR}/_output/rpmbuild-fake-next-minor"
 
 # Location of local repository used by composer
 # shellcheck disable=SC2034  # used elsewhere
-LOCAL_REPO="${IMAGEDIR}/rpm-repos/microshift-local"
+SUBST_LOCAL_REPO="${IMAGEDIR}/rpm-repos/microshift-local"
 
 # Location of local repository used by composer
 # shellcheck disable=SC2034  # used elsewhere
-NEXT_REPO="${IMAGEDIR}/rpm-repos/microshift-fake-next-minor"
+SUBST_NEXT_REPO="${IMAGEDIR}/rpm-repos/microshift-fake-next-minor"
```

Comment:
This file, at least, should have "normal" variable names only. Scripts might need to use the values.

Reply:
OK, I reverted the commit.